sdk/security/keyvault/azadmin/backup/constants.go

```diff
@@ -0,0 +1,10 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+// DO NOT EDIT.
+
+package backup
```

Comment:
This file can be deleted.  If the code generator is creating any empty file, that's a bug.

Reply:
Confirmed codegen bug, https://github.com/Azure/autorest.go/pull/937

Reply:
deleted in 721f245